source/docs/software/frc-glossary.rst

```diff
@@ -59,6 +59,9 @@ FRC Glossary
    NetworkTables
       A way to communicate key / value pairs, of data, between programs.
 
+   non-linear
+      In a control system: some system which is not well-modeled by a set of polynomials and their derivatives. For example, in a mechanism where friction forces jump instantaneously between two values (due to changing static/kinetic coefficients of friction), that mechanism can be said to act in a "non-linear" way.
```

Comment:
I'm a bit surprised that the term is used in one context, and then explained in the glossary with a totally different example that isn't mentioned anywhere else

Reply:
We should probably remove this.  Nonlinearity is pretty complicated and probably not within-scope; we should keep all of this implicitly within the realm of linear control theory.  We can talk about linearization in a separate article maybe, but that seems to be getting towards the realm of Tyler's book so that might be out of scope

Reply:
It's worth noting that the meaning of the term "linear" in the context of controls is "algebraically homomorphic," which is probably not something that *most* students are quite ready to deal with in high school.  The term is unfortunately quite overloaded.

Reply:
Definitely ok with removing it. "Nonlinearity" here was brought up as a proxy for saying 'This is purposefully and example where PID by itself performs poorly". Rather than getting technical, I'll reword to just express that notion, and maybe point to a section of the textbook so a student who likes depth-first knowledge searches can get a taste as to "why" it would perform poorly.